osia/installer/downloader/install.py

```diff
@@ -149,12 +169,21 @@ def download_installer(installer_version: str,
     else:
         raise Exception("Error for source profile " + source)
 
-    url, version = downloader(installer_version, installer_arch)
+    if fips and Version.coerce(installer_version.split("-")[-1]) in SimpleSpec("<4.16"):
+        # ocp <4.16 does not have dedicated openshift-install-fips,
+        # it just need to be run on fips enabled RHEL
```

Comment:
isn't `openshift-baremental-install` needed for proper fips install of 4.12? That was my understanding from OCP team some time ago, but maybe I misunderstood them.

Reply:
I did not found any fips related documentation regarding "proper" installer. I compared: 
https://docs.openshift.com/container-platform/4.15/installing/overview/installing-fips.html
https://docs.openshift.com/container-platform/4.16/installing/overview/installing-fips.html

Only 4.16+ includes "Obtaining a FIPS-capable installation program" and says all `openshift-install` should be changed to `openshift-install-fips` in further documentation.

Reply:
I think `openshift-baremetal-install` exists only for 4.12 as 4.12 is fips designed or something like that. TBH not sure what is the status now. This was my last info from openshift team several weeks ago.

Reply:
`openshift-baremetal-install` does not work as replacement for `openshift-install`
```
osia.installer.downloader.image.CoreOsException: (CoreOsException(...), "Installer doesn't support coreos subcommand")
```

Reply:
Strange....guess another change from OCP team 🤷